src/js/components/Meter.js

```diff
@@ -683,6 +692,134 @@ var Meter = React.createClass({
     );
   },
 
+  _renderIndicators: function (type, id) {
+    //Render high and average indicator for all values
+    var start;
+    var commands;
+    var classes;
+    var angle;
+    var endangle;
+    var radio;
+    var indicator;
+    var index;
+    if (id === "high") {
+      indicator = this.state.highIndicator;
+      index = 1;
+    } else if (id === "average") {
+      indicator = this.state.averageIndicator;
+      index = 2;
+    }
+    if(this.props.type === "bar") {
+      classes = [CLASS_ROOT + "__bar", CLASS_ROOT + "__bar--active", "color-index-" + type];
+      start = this._translateBarWidth(indicator);
+      commands = this._barCommands(start, 1);
+    } else if (this.props.type === "arc" || this.props.type === "circle") {
+      angle = this.state.startAngle;
+      classes = [CLASS_ROOT + "__slice", CLASS_ROOT + "__slice--active", "color-index-" + type];
+      endangle = this._translateEndAngle(angle, indicator);
+      start = this._translateEndAngle(angle, indicator - 1);
+      commands = this._arcCommands(start, endangle);
+    } else if (this.props.type === "spiral") {
+      angle = this.state.startAngle;
+      classes = [CLASS_ROOT + "__slice", CLASS_ROOT + "__slice--active", "color-index-" + type];
+      endangle = this._translateEndAngle(angle, indicator);
+      start = this._translateEndAngle(angle, indicator - 1);
+      radio = this.state.startRadius;
+      commands = this._spiralCommands(start, endangle, radio);
+    }
+    return this._buildPath(commands, null, null, index, classes);
+  },
+
+  _maxValue: function() {
+    //Get high value for high indicator
+    var high = this._seriesMax(this.state.series);
+    if (high > this.state.highIndicator) {
+      this.state.highIndicator = high;
+      this.state.date1 = this._createDate("/");
+    }
+  },
+
+  _sumValues:function(e) {
+    //Acumulated values for average values
+    var sum = 0;
+    var self = this;
+    e.map(function(data, i) {
+      sum += data.value;
+      self.state.averageCounter += 1;
+    });
+    return sum;
+  },
+
+  _averageValue: function() {
+    //Get average value for average indicator
+    if (this.state.mouse === false) {
+      this.state.averageSum += this._sumValues(this.state.series);
+      this.state.averageIndicator = this.state.averageSum / this.state.averageCounter;
+      this.state.date2 = this._createDate("/");
+    }
+  },
+
+  _createDate: function(c){
```

Comment:
can we use a function that already gives us formatteddate? This will not work specially around Internationalization.